modules/core/src/main/scala/org/scalasteward/core/dependency/DependencyService.scala

```diff
@@ -58,7 +59,7 @@ class DependencyService[F[_]](
       _ <- logger.info(s"Refresh dependencies of ${repo.show}")
       cloneUrl = util.uri.withUserInfo(repoOut.clone_url, config.gitHubLogin)
       _ <- gitAlg.clone(repo, cloneUrl)
-      parent <- repoOut.parentOrRaise[F]
+      parent <- repoOut.parentOrRaise[F](config)
       parentCloneUrl = util.uri.withUserInfo(parent.clone_url, config.gitHubLogin)
       _ <- gitAlg.syncFork(repo, parentCloneUrl, parent.default_branch)
```

Comment:
In https://github.com/fthomas/scala-steward/pull/208/files#diff-bdb025c4edf5aaff2f7f02c1a209ad42R65 `syncFork` is guarded with `doNotFork`. Why not here?

Reply:
Thanks! Added a guard.